### PR TITLE
added input component

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**

### CODE REVIEW

File: Stylesheets/components/input.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    font-family: 'Montserrat', sans-serif;
+    
+}
+
+.flex{
+    display: flex;
+    flex-direction: column;
+    width:10rem;
+    margin-bottom: 1rem;
+    width:30rem
```
Comment (srejitk) [inline Stylesheets/components/input.css:10]: You have mentioned width twice here. If you intended max width here , please correct that here.

Comment (srejitk) [inline Stylesheets/components/input.css]: The class name here is ambiguous. Extract the common properties to a utils class and rename it as required.

File: Stylesheets/components/input.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    
+
+}
+
+.name-input, .password-input{
+    display:block;
+    border-radius:0.2rem;
+    font-size:1rem;
+    font-weight:bold;
+    border:0.15rem solid #03045e;
```
Comment (srejitk) [inline Stylesheets/components/input.css]: Would suggest assigning this color to a variable.

File: Stylesheets/components/input.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    margin-bottom: 1rem;
+    
+}
+
+.val-input{
+    display:block;
+    border-radius:0.2rem;
+    font-size:1rem;
+    font-weight:bold;
+    border:0.15rem solid green;
```
Comment (srejitk) [inline Stylesheets/components/input.css]: Here too. Replace with Variables to maintain uniformity.

File: Stylesheets/components/input.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+.val-password{
+    display:block;
+    border-radius:0.2rem;
+    font-size:1rem;
+    font-weight:bold;
+    border:0.15rem solid var(--red-color);
+    width:20rem;
+    
+}
```
Comment (srejitk) [inline Stylesheets/components/input.css]: Except the border color, All the other properties are common. You can extract it to another class and simplify the code.

File: Stylesheets/components/input.css [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    margin-bottom: 1rem;
+}
+span{
+    padding-left: 0.3rem;
+}
+
+.flex-col{
+    display: flex;
+    flex-direction: column;
+}
```
Comment (srejitk) [inline Stylesheets/components/input.css]: Move this to the utils class.

**Review by srejitk — COMMENTED** [review]
Suggested a few modifications.

**Review by kalyandindu — COMMENTED** [review]
I would suggest to create a design system for typography, spacing, borders etc., for consistency. Also convert commonly used styles into util classes for faster development.

File: Stylesheets/components/input.css (hunk 4 above)
Comment (Priyanshi-Tiwari) [reply Stylesheets/components/input.css]: done...thanks

File: Stylesheets/components/input.css (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply Stylesheets/components/input.css]: done

**Pull request merged**